js/oop/Factorial.js

```diff
@@ -12,6 +12,23 @@ export class Factorial {
   }
 
   // write your code here
+  loop() {
```

Comment:
Please consider a better name for these two functions (loop and recursive) e.g. countUsingLoop, etc. You always have to make sure that your method name is describing what you actually do on that method.